container-engine-lib/lib/backend_impls/docker/docker_manager/docker_manager.go

```diff
@@ -1728,7 +1728,15 @@ func (manager *DockerManager) getContainersByFilterArgs(ctx context.Context, fil
 	if err != nil {
 		return nil, stacktrace.Propagate(err, "An error occurred getting the docker containers with filter args '%+v'", filterArgs)
 	}
-	containers, err := newContainersListFromDockerContainersList(dockerContainers)
+	dockerContainersDetails := []types.ContainerJSON{}
+	for _, dockerContainer := range dockerContainers {
+		dockerContainerDetails, err := manager.InspectContainer(ctx, dockerContainer.ID)
```

Comment:
Switch to the Docker container inspect to get the information we need.  Calls to the Docker API are fast and I don't expect a performance hit.